tests/test_code_report.py

```diff
@@ -76,3 +77,18 @@ def test_pylint_analyzer_wrong_params(runner_with_pylint, args, expected_log):
     log = runner_with_pylint.run(get_config(args))
     assert re.findall(r'Run static pylint - [^\n]*Failed', log)
     assert expected_log in log
+
+
+def test_code_report_extended_arg_search(runner_with_pylint):
```

Comment:
This test needs some comments.
The portion with expanding the "variable" in artifacts is ok, but with bash is it not clear enough.
BTW, why it didn't work with bash?

Reply:
Because this exact issue is outdated, and some older versions of code searched for fake variable in `cmd`, not in every `cmd` member. It was then fixed, but the issue was however not closed.

Reply:
So, if I remove the reference to the variable from artifacts key and revert the fix, would the test succeed?

Reply:
Sure.

Reply:
Final question: when we fixed the issue with bash?